src/js/helpers/searchHelper.js

```diff
@@ -178,3 +247,38 @@ export const performFinancialSystemLookup = (params) => {
         }
     };
 };
+
+// Budget Filters consist of the three elements of Budget Categories filter
+// (Budget Functions, Federal Account, and Object Class),
+// as well as Fiscal Year filters and Funding Agency
+export const budgetFiltersSelected = (params) => {
+    if (params.budgetFunctions.size > 0
```

Comment:
I am confident there is a way to perform these checks using some kind of loop instead of a monster if statement (also applies to `awardFiltersSelected`)